compose/config/fields_schema_v2.json

```diff
@@ -30,6 +39,10 @@
   },
 
   "definitions": {
+    "network": {
+      "id": "#/definitions/network",
+      "type": "object"
+    },
```

Comment:
I assume this is temporary and the keys will be defined when the alias support is flushed out?